### ISSUE TITLE
Automatic use first image on slider

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/zillah#165]
A lot of people assume that if they choose the option to automatic pick featured image, the slider will do that too. So either we should indeed do it or make this information clear in slider control so people don't get confused.

What say? @rodica-andronache

**Comment by rodica-andronache** [comment]
yes, I see there is a lot of confusion with this. @ineagu should we change this for the slider too, if possible?

**Comment by ineagu** [comment]
yep
